edx-platform/pearson-regis-theme/lms/static/sass/partials/lms/theme/_dashboard.scss

```
@@ -0,0 +1,463 @@
+// Change cards from landscape to portrait
+
+.view-dashboard div.item .items.course.single-course {
+  float: left;
+  margin: 1%;
+  min-width: 250px;
+  padding: 0px;
+  width: 23%;
+}
+
+@media (max-width: 1190px) {
+  .view-dashboard div.item .items.course.single-course {
+    width: 31%;
+  }
+}
+
+@media (max-width: 890px) {
+  .view-dashboard div.item .items.course.single-course {
+    width: 48%;
+  }
+}
+
+@media (max-width: 660px) {
+  .view-dashboard div.item .items.course.single-course {
+  width: 98%;
+  }
+}
+
```

```suggestion
.view-dashboard div.item .items.course.single-course {
  float: left;
  margin: 1%;
  min-width: 250px;
  padding: 0;
  width: 23%;

  @include media-breakpoint-down(lg) {
    width: 31%;
  }

  @include media-breakpoint-down(md) {
    width: 49%;
  }

  @include media-breakpoint-down(sm) { 
    width: 98%;
  }
}
```

Would you consider to nest the media queries inside the selector to avoid repetition? And also use Bootstrap's breakpoints instead of the magic methods